packages/core-typings/src/IInquiry.ts

```diff
@@ -33,16 +34,16 @@ export interface ILivechatInquiryRecord extends IRocketChatRecord {
 	ts: Date;
 	message: string;
 	status: LivechatInquiryStatus;
-	v: IVisitor;
+	v: Pick<ILivechatVisitor, '_id' | 'username' | 'status' | 'name' | 'token' | 'phone'> & { lastMessageTs?: Date };
 	t: 'l';
 
-	department: string;
-	estimatedInactivityCloseTimeAt: Date;
+	department?: string;
+	estimatedInactivityCloseTimeAt?: Date;
 	locked?: boolean;
 	lockedAt?: Date;
 	lastMessage?: IMessage & { token?: string };
 	defaultAgent?: SelectedAgent;
-	source: {
+	source?: {
```

Comment:
Can source be optional?

Reply:
It does have a default option for `others` so perhaps we can use it as default

Reply:
It "can" be optional, but was required since forever, so at this point, probably all inquiries have a source. So, "not optional" for now